.fttemplates/TS Functional Component/[FTName].tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+import styles from './[FTName].styles';
+
+const FTName: React.FunctionComponent = () => (
```

Comment:
I've recently seen components typed this way more often
```suggestion
interface <FTName>Props = {}

const FTName= ({}: <FTName>Props) => (
```
Depending on the version of react that react native is using `React.FunctionComponent` [implicitly receives children as props by default and we might not want that for every component.](https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13#:~:text=FunctionComponent%20provides%20an%20implicit%20definition,children%20provided%20to%20your%20component.)

I don't see a benefit to adding `React.FunctionComponent` but I am open to discussing this and come up with the optimal solution